### PR TITLE
GitHub: Improved infrastructure of win32metadata

### PR DESCRIPTION
**Pull request by 0x5bfa**
## Summary

- Updated README.md to have a header and simplify links
- Added issue templates for bug report and question
- Added links to go CsWin32 and windows-rs before submitting to a wrong place
- Disabled blank issue
- Added pull request template (the same this this)

## Checklist

- [ ] Closes #xxx
- [ ] Built to validate changes

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## Summary
+
+
+
+## Checklist
+
+- [ ] Closes #xxx
+- [ ] Built to validate chnages
```
Comment (getrou) [inline .github/PULL_REQUEST_TEMPLATE.md:8]: typo - changes

Comment (0x5bfa) [reply .github/PULL_REQUEST_TEMPLATE.md]: Thanks, fixed.

**Comment by riverar** [comment]
Not sure about these changes, especially those that create more overhead in our issues/PR workflows (e.g., like requiring we check some box indicating we've tested locally).

**Review by riverar — CHANGES_REQUESTED** [review]
Not sure about these changes, especially those that create more overhead in our issues/PR workflows (e.g., like requiring we check some box indicating we've tested locally).

I think we can limit the changes to doc tweaks and `.github` consolidation, but we'll need to double check the backend doesn't have any dependencies there.

**Comment by 0x5bfa** [comment]
I deleted PR template. Yet honestly I'd find issue templates very useful, what do you think? If not needed, I'd delete them.

I added the clear header in the readme and tweaked some.

**Pull request closed without merging**